src/Logger/LogsHttpLogger.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\logs_http\Logger\LogsHttpLogger.
+ */
+
+namespace Drupal\logs_http\Logger;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Logger\LogMessageParserInterface;
+use Drupal\Core\Logger\RfcLogLevel;
+use Drupal\Core\Logger\RfcLoggerTrait;
+use Psr\Log\LoggerInterface;
+
+class LogsHttpLogger implements LoggerInterface {
+  use RfcLoggerTrait;
+
+  /**
+   * A configuration object containing logs_http settings.
+   *
+   * @var \Drupal\Core\Config\Config
+   */
+  protected static $config;
+
+  /**
+   * The message's placeholders parser.
+   *
+   * @var \Drupal\Core\Logger\LogMessageParserInterface
+   */
+  protected $parser;
```

Comment:
=> `$logMessageParser` (always follow the class name, minus the `interface` part)